metrics-proxy/pom.xml

```diff
@@ -149,6 +149,11 @@
       <artifactId>hamcrest-core</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
```

Comment:
We try to avoid using mocking libraries, so please consider if there is an alternative approach to testing.